application/features/sync.feature

```diff
@@ -26,3 +26,10 @@ Feature: Synchronizing a record
       And the user does not exist in the ID Broker
     When I learn the user does not exist in the ID Store and I tell the ID Broker
     Then the ID Broker response should return an error
+
+  Scenario: User info in ID Broker does not equal user info in ID Store
+    Given the user exists in the ID Store
+      And the user exists in the ID Broker
+      And the user info in the ID Broker does not equal the user info in the ID Store
+    When I send the user info in the ID Store to the ID Broker
```

Comment:
I prefer the...
`When I get the user info from the ID Store and send it to the ID Broker`
... wording over the...
` When I send the user info in the ID Store to the ID Broker`
... wording because this is the ID Sync, and it can't _send_ anything _from_ the ID Store. It can get info from the ID Store and then send it to the ID Broker, but the ID Sync has to initiate the retrieval. Even for the ad-hoc situation, the ID Sync is only given an employee ID. It then has to ask the ID Store for the actual info about that user.

I guess my main concern is keeping the perspective correct for clarity's sake. Thoughts?